keyboards/mlego/m48/rev1/config.h

```diff
@@ -56,3 +56,6 @@
 #define RGBLIGHT_EFFECT_RGB_TEST
 #define RGBLIGHT_EFFECT_ALTERNATING
 #define RGBLIGHT_EFFECT_TWINKLE
+
+// you want to comment this if using stm32-dfu as bootloader
+#define FEE_PAGE_BASE_ADDRESS 0x0800c000
```

Comment:
I'm not sure that this works? 

Have you confirmed that it actually does?

Reply:
you are right is not even compiling once I activate via...

Reply:
ok fixed and confirmed with dasky's changes custom ld.